themes/theme-b2b/src/components/NumberField.styles.ts

```diff
@@ -2,18 +2,18 @@ import { ThemeComponent, cva } from '@marigold/system';
 
 export const NumberField: ThemeComponent<'NumberField'> = {
   group: cva([
-    'border-border-light rounded-sm border border-solid',
-    'group-hover/field:border-border-hover',
-    'group-readonly/field:group-focus/field:border-border-light group-readonly/field:group-focus/field:shadow-none',
-    'group-disabled/field:bg-bg-disabled group-disabled/field:text-text-disabled group-disabled/field:cursor-not-allowed ',
+    'border-border-base rounded-sm border border-solid',
+    'group-hover/field:border-border-base-hover',
+    'group-readonly/field:group-focus/field:border-border-base group-readonly/field:group-focus/field:shadow-none',
+    'group-disabled/field:bg-bg-base-disabled group-disabled/field:text-text-base-disabled group-disabled/field:cursor-not-allowed ',
     'data-[focus-within]:outline-outline-focus data-[focus-within]:outline',
     'outline-2 -outline-offset-1 ',
-    'group-error/field:outline-outline-error group-error/field:outline',
+    'group-error/field:border-border-error ',
   ]),
   stepper: cva([
-    'text-text-body w-7',
-    'border-border-light border-solid first-of-type:border-r',
-    'border-border-light border-solid last-of-type:border-l ',
+    'text-text-base w-7',
+    'border-border-base border-solid first-of-type:border-r',
```

Comment:
I'm unable to find this line of code anywhere in the component.

Reply:
So some of our components are divided into parts, and these parts can be styled sperat. So in this case the Numberfield contains multiple parts which can be styled. This line of code is from the `stepper` part. It's a bit difficult to spot it. The stepper are the plus and minus buttons.

Reply:
Alles claro